packages/host/app/services/matrix-service.ts

```diff
@@ -52,11 +54,35 @@ export type OperatorModeContext = {
   openCards: CardDef[];
 };
 
+function onNameChange(service: MatrixService) {
+  return (event: MatrixEvent, member: RoomMember) => {
+    console.log('event', event, member);
+    // FIXME isn’t this a questionable place for this?
```

Comment:
The other event handlers are extracted into files in `lib/matrix-handlers` and receive the service as a `Context`, probably that makes sense here too.